dev-infrastructure/global-pipeline.yaml

```diff
@@ -1,3 +1,12 @@
+#
+# Purpose: Manage global infrastructure for ARO HCP
```

Comment:
Could add a required `description` field and compile it into an additional document instead of comments. WDYT?